examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light",
+      "tags": []
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Calculating predictions of the trained network on a batch\n",
+    "outputs = torch.tensor(model.predict_on_batch(inputs))\n",
+    "\n",
+    "output_grid = make_grid(outputs)\n",
+    "fig = plt.figure(figsize=(10, 10))\n",
+    "out = output_grid.numpy().transpose((1, 2, 0))\n",
+    "plt.imshow(out)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "e4kdBqMsw_P5"
+   },
+   "source": [
+    "The code below show how we can takte advantages of poutyne to resume training from a special epoch (for example if we found the past epoch not enough, we can resume from the last done epoch, as below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "9I-cfFgFw_P5",
+    "outputId": "67002074-b771-4653-f9ff-8d60840fa972"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch: 5/7 Step: 1500/1500 100.00% |████████████████████|22.13s loss: 0.019525 val_loss: 0.019379\n",
+      "Epoch 5: val_loss improved from 0.01980 to 0.01938, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 6/7 Step: 1500/1500 100.00% |████████████████████|21.99s loss: 0.019125 val_loss: 0.019044\n",
+      "Epoch 6: val_loss improved from 0.01938 to 0.01904, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 7/7 Step: 1500/1500 100.00% |████████████████████|21.24s loss: 0.018845 val_loss: 0.018818\n",
+      "Epoch 7: val_loss improved from 0.01904 to 0.01882, saving file to saves/best_weight.ckpt\n",
+      "Restoring model from saves/best_weight.ckpt\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[{'epoch': 5,\n",
+       "  'loss': 0.019525130399813254,\n",
+       "  'time': 22.13099593600009,\n",
+       "  'val_loss': 0.01937860634426276},\n",
+       " {'epoch': 6,\n",
+       "  'loss': 0.01912545581596593,\n",
+       "  'time': 21.99487811600011,\n",
+       "  'val_loss': 0.019044068900247416},\n",
+       " {'epoch': 7,\n",
+       "  'loss': 0.018844680001338324,\n",
+       "  'time': 21.23843467100005,\n",
+       "  'val_loss': 0.018818224750459193}]"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {
+      "tags": []
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.fit_generator(trn_dataloader, val_dataloader, epochs=7, callbacks=callbacks, initial_epoch=num_epochs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sxcPMY6Nw_P5"
+   },
+   "source": [
+    "# Reconstructed images after the second training process"
```

Comment:
```suggestion
    "## Reconstructed images after the second training process"
```

Reply:
I think one could have a fig with both the prediction (previous training and this one) to compare next to each other.